compact/tracking/mpgd_outerbarrel.xml

```diff
@@ -13,20 +13,21 @@
 
 <define>
     <comment> Frames </comment>
-    <constant name="MPGDOuterBarrelFrame_width"            value="10*mm"/>
+    <constant name="MPGDOuterBarrelFrame_width"            value="15*mm"/>
     <constant name="MPGDOuterBarrelFrame_thickness"        value="7*mm"/>
 
     <comment> Module constants </comment>
     <constant name="MPGDOuterBarrelModule_roverlap"                value="0*cm"/>
     <constant name="MPGDOuterBarrelModule_zoverlap"                value="0*MPGDOuterBarrelFrame_width"/>
     <constant name="MPGDOuterBarrelModule_count"                   value="12" />
-    <constant name="MPGDOuterBarrelModule_allowed_space"           value="2.5*cm"/>
+    <constant name="MPGDOuterBarrelModule_allowed_space"           value="25*mm"/>
     <constant name="MPGDOuterBarrelModule_rmax"                    value="MPGDOuterBarrelModule_rmin + MPGDOuterBarrelModule_allowed_space" />
     <constant name="MPGDOuterBarrelModule_roffset"                 value="0.5*cm" />
-    <constant name="MPGDOuterBarrelModule_width"                   value="36.0*cm"/>
+    <constant name="MPGDOuterBarrelModule_width"                   value="360*mm"/>
     <constant name="MPGDOuterBarrel_length"                        value="MPGDOuterBarrelModule_zmin1 + MPGDOuterBarrelModule_zmin2"/>
     <constant name="MPGDOuterBarrelModule_length"                  value="0.5*(MPGDOuterBarrelModule_zmin1 + MPGDOuterBarrelModule_zmin2 + MPGDOuterBarrelModule_zoverlap)"/>
-    <constant name="MPGDOuterBarrelModule_offset"                  value="(MPGDOuterBarrelModule_zmin2 - MPGDOuterBarrelModule_zmin1)/2.0"/>
+    <constant name="MPGDOuterBarrelModule_offset"                  value="(MPGDOuterBarrelModule_zmin2 - MPGDOuterBarrelModule_zmin1)"/>
```

Comment:
This is not how offset is usually defined. Why no div by 2? Normally it's the half the difference since that is where the center of the volume is compared to the origin. Can you add a comment here?

```
    <constant name="MPGDOuterBarrelModule_zmin1"     value="1795*mm"/>
    <constant name="MPGDOuterBarrelModule_zmin2"     value="1845*mm"/>
```
This means that the barrel extends from `-1795` to `+1845`. That means the center is at `+25`, not at `+50`.